crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/load.rs

```diff
@@ -43,7 +44,21 @@ pub fn load(
 /// For details see:
 /// <https://docs.starknet.io/documentation/architecture_and_concepts/Smart_Contracts/contract-storage>
 #[must_use]
-pub fn map_storage_address(address: FieldElement) -> FieldElement {
+fn normalize_storage_address(address: FieldElement) -> FieldElement {
     let modulus = Felt252::from(2).pow(251) - Felt252::from(256);
     address % modulus.into_()
 }
+
+#[must_use]
+pub fn calculate_variable_address(selector: Felt252, key: Option<&[Felt252]>) -> Felt252 {
```

Comment:
Could this function take in variable name instead of selector? It seems every use needs to be preceded by calculating the selector

Reply:
This is going to be calculated in cairo via the `selector!` macro so that's why i decided to converge towards that interface

Reply:
Maybe another function then? 👀

Reply:
it would be written purely for tests, so i can perhaps put it in there if you'd like that solution more

Reply:
I would like that 👍